Api/Model/Action/Export.php

```diff
@@ -334,89 +328,83 @@ private function _getGiftMessageInfo($messageContainer): self
             $this->addXmlElement("GiftMessage", "<![CDATA[From: {$gift->getSender()}\nTo: {$gift->getRecipient()}\nMessage: {$gift->getMessage()}]]>");
         }
 
-        $this->addXmlElement("Gift", !is_null($giftId));
+        $this->addXmlElement("Gift", !is_null($giftId) ? 'true' : 'false');
```

Comment:
Will $giftId ever be 0 or an empty string? If so, I think is_null() is going to return false in those cases and potentially give an unexpected result here.